src/os_mswin.c

```diff
@@ -1860,7 +1860,7 @@ typedef BOOL (WINAPI *pfnGetFileInformationByHandleEx)(
 	FILE_INFO_BY_HANDLE_CLASS_	FileInformationClass,
 	LPVOID				lpFileInformation,
 	DWORD				dwBufferSize);
-static pfnGetFileInformationByHandleEx pGetFileInformationByHandleEx = NULL;
```

Comment:
Isn't it better to initialize the variable explicitly for readability?
(I understand that global and static variables are initialized by 0.)
Also the same for `loaded`.